server/models/schema.sql

```diff
@@ -628,6 +628,7 @@ CREATE TABLE `enterprise_setting` (
   `enable_delete_records` TINYINT(1) NOT NULL DEFAULT 0,
   `enable_prepayments` TINYINT(1) NOT NULL DEFAULT 1,
   `enable_password_validation` TINYINT(1) NOT NULL DEFAULT 1,
+  `enable_prepayment_inside_invoice` TINYINT(1) NOT NULL DEFAULT 0,
```

Comment:
This is the most important label to change (since it will be the most difficult to change later).  Let's call it `enable_balance_on_invoice_receipt`.